array-manipulation-2/src/drop-right.js

```diff
@@ -0,0 +1,11 @@
+/* exported dropRight */
+function dropRight(array, count) {
+  var newArray = [];
+  for (var i = 0; i < array.length - count; i++) {
+    var value = array[i];
+    if (value || typeof value === 'boolean') {
```

Comment:
I don't think this conditional is necessary. If this is to deal with the array being shorter than the length requested, I would use a conditional before the loop to check if the count is larger than the length, and if it is, just return an empty array